app/models/repository.js

```diff
@@ -1,8 +1,19 @@
+import Ember from 'ember';
 import DS from 'ember-data';
 
 export default DS.Model.extend({
   name: DS.attr('string'),
-  slug: DS.attr('string'),
   description: DS.attr('string'),
-  namespace: DS.belongsTo()
+  namespace: DS.belongsTo(),
+  "content_type": DS.attr('string'),
+  "public_access": DS.attr('boolean'),
+
+  repoSlug: Ember.computed('id', function() {
```

Comment:
This should be called `repoId` as well (see the other comment on `splitRepoSlug`).